.gitignore

```diff
@@ -28,6 +28,7 @@ bld/
 .vs/
 # Uncomment if you have tasks that create the project's static files in wwwroot
 #wwwroot/
+Migrations/
```

Comment:
We find it easier to delete and rebuild the database from scratch when changing between different pull requests.